rero/rero-ils-ui
# request: implement access to request queue from modal

* Corrects private variables to follow project syntax.
* Adds number of requests on top of modal window if there is a queue.
* Implements access to queue in item detail view from modal.
* Removes weird character in licence text.

Co-Authored-by: Alicia Zangger <alicia.zangger@rero.ch>

## Why are you opening this PR?

https://tree.taiga.io/project/rero21-reroils/task/1418?kanban-status=1224894
To implement access to request queue in item detail view from 'add request' modal window.

## How to test?

Go to a document detail view with request(s).
Click on 'request' button for an item.
A modal window with a list of pending request (and 'add request' form) should be displayed:
![modal](https://user-images.githubusercontent.com/28982829/78371080-c73fa480-75c7-11ea-9f86-615bc0030aaf.png)

Click on 'Edit queue' button. You should be redirected to the item detailed view.

## Code review check list
